newsroom/core/resources/service.py

```diff
@@ -34,3 +34,10 @@ async def on_deleted(self, doc: NewshubResourceModelType):
         if self.clear_item_cache_on_update:
             app = get_current_wsgi_app()
             app.cache.delete(str(doc.id))
+
+    async def find_items_by_ids(self, ids: list[str]) -> list[NewshubResourceModelType]:
```

Comment:
This would be best off being in the core AsyncResourceService, as this is not Newshub specific

Reply:
I'll add the method in separate PR in SD core and here too.